src/js/streams.ts

```diff
@@ -276,6 +276,17 @@ function setDefaultStdout() {
  * @param options.isatty Should :py:func:`isatty(stdout) <os.isatty>` return
  * ``true`` or ``false``. Can only be set to ``true`` if a raw handler is
  * provided (default ``false``).
+ * @example
+ * async function main(){
+ *   const pyodide = await loadPyodide();
+ *   pyodide.setStdout({ batched: (msg) => console.log(msg) });
+ *   pyodide.runPython("print('ABC')");
+ *   // 'ABC'
+ *   pyodide.setStdout({ raw: (byte) => console.log(byte) });
+ *   pyodide.runPython("print('ABC')");
+ *   // 65 66 67 10 (ascii values for 'ABC' including a new line character)
```

Comment:
Since it prints them all on separate lines, maybe:
```suggestion
 *   // 65
 *   // 66 
 *   // 67 
 *   // 10 (the ascii values for 'ABC' including a new line character)
```